code/modules/antagonists/traitor/objectives/destroy_item.dm

```diff
@@ -34,6 +34,28 @@
 		/datum/objective_item/steal/blackbox,
 	)
 
+/// Super early-game destroy objective intended to be items easily tided that the crew tends to value.
+/datum/traitor_objective/destroy_item/demoralise
+	description = "Find %ITEM% and destroy it using any means necessary. \
+		We believe this luxury item is important for crew morale. \
+		Destruction of this item will help our recruitment efforts."
+
+	progression_minimum = 0 MINUTES
+	progression_maximum = 10 MINUTES
+	progression_reward = list(4 MINUTES, 8 MINUTES)
+	telecrystal_reward = list(0, 1)
+
+	possible_items = list(
+		/datum/objective_item/steal/traitor/donut_box,
+		/datum/objective_item/steal/traitor/rpd,
+		/datum/objective_item/steal/traitor/space_law,
+		/datum/objective_item/steal/traitor/granted_stamp,
+		/datum/objective_item/steal/traitor/denied_stamp,
+		/datum/objective_item/steal/traitor/lizard_plush,
+		/datum/objective_item/steal/traitor/moth_plush,
+		/datum/objective_item/steal/traitor/insuls,
+	)
```

Comment:
To be honest, some of these feel like they have the same issues as the poster objective in that this is a filler objective.
The insulated gloves are cool, but I think you can remove most of the trivial ones like the moth, lizard plush and stamps

Reply:
I can do, but then it leaves this objective pretty barebones as it just has RPD, space law book and insuls. Any suggested replacements? Also see my comment way below about why I picked some of these items and my goal with this objective in general.

Reply:
I'd say remove the donut box since it can be found in public areas, but you can keep the rest I suppose.